### PR TITLE
Remove json5 dependency

### PR DESCRIPTION
**Pull request by A-312**
(break change)

Fix: #320

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +47,7 @@
     "deep-object-diff": "1.1.0",
     "eslint": "5.16.0",
     "js-yaml": "3.13.1",
+    "json5": "^2.1.0",
```
Comment (madarche) [inline package.json]: Please use only exact versions, cf. https://github.com/mozilla/node-convict/blob/master/CONTRIBUTING.md#dependencies

**Comment by madarche** [comment]
Apart for the small comment about the version, I'm all for this PR. Thanks.

**Comment by A-312** [comment]
Can you make the change ? Or i will next week

**Comment by madarche** [comment]
@A-312 Since this change is a major change, I'll first release a v5.1.1 with the previous changes. I'll try to do the change myself, but If I haven't had the chance to do this by Monday, please do it yourself.

**Comment by A-312** [comment]
Too many conflict with package-lock.json, I thought that will be easier.

 @madarche This pr is ready

**Comment by madarche** [comment]
Thanks @A-312 for this useful PR which goes in the direction of simplifying convict.

**Comment by hermanbanken** [comment]
Maybe add to the README how to do that to see the breaking change migration? @madarche

**Comment by madarche** [comment]
@hermanbanken Good 😁

**Pull request merged**